charts/scalardl-audit/templates/auditor/deployment.yaml

```diff
@@ -70,26 +78,31 @@ spec:
                 name: {{ include "scalardl-audit.fullname" . }}-auditor
               {{- end }}
                 key: db-password
-          - name: SCALAR_DB_STORAGE
+          - name: HELM_SCALAR_DB_STORAGE
             value: "{{ .Values.auditor.scalarAuditorConfiguration.dbStorage }}"
-          - name: SCALAR_DL_AUDITOR_SERVER_PORT
+          - name: HELM_SCALAR_DL_AUDITOR_SERVER_PORT
             value: "{{ .Values.auditor.scalarAuditorConfiguration.auditorServerPort }}"
-          - name: SCALAR_DL_AUDITOR_SERVER_PRIVILEGED_PORT
+          - name: HELM_SCALAR_DL_AUDITOR_SERVER_PRIVILEGED_PORT
             value: "{{ .Values.auditor.scalarAuditorConfiguration.auditorServerPrivilegedPort }}"
-          - name: SCALAR_DL_AUDITOR_SERVER_ADMIN_PORT
+          - name: HELM_SCALAR_DL_AUDITOR_SERVER_ADMIN_PORT
             value: "{{ .Values.auditor.scalarAuditorConfiguration.auditorServerAdminPort }}"
           - name: SCALAR_DL_AUDITOR_LOG_LEVEL
```

Comment:
We cannot add `HELM_` prefix to `SCALAR_DL_AUDITOR_LOG_LEVEL` since this is used in the `log4j2.properties` in the container and we cannot override `log4j2.properties` using Helm Charts at this time.